src/ghdcbot/engine/thread_to_issue.py

```
@@ -0,0 +1,599 @@
+        section_fills.append(
+            (r"(###?\s*(?:Environment|System\s*Info)[^\n]*\n)", f"\\1\n{env_info}\n")
+        )
+
+    if log_snippets:
+        logs_md = "\n\n".join(f"```\n{s}\n```" for s in log_snippets[:3])
+        section_fills.append(
+            (r"(###?\s*(?:Logs?|Error\s*(?:Output|Log))[^\n]*\n)", f"\\1\n{logs_md}\n")
+        )
+
+    for pattern, replacement in section_fills:
+        result, count = re.subn(pattern, replacement, result, count=1, flags=re.IGNORECASE)
```

Ruff (0.16.1)

[warning] Unpacked variable `count` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)